src/konnektor/tests/network_planners/_networkx_implemenations/conftest.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+
+
+@pytest.fixture(scope="session")
```

Comment:
I am assuming these never get mutated?

Reply:
correct, but I also don't like session scoped things unless totally necessary. I dont think there's anything to warrant a session scope here (complex or expensive setup), so i'm taking it out!

Reply:
p.s. there's a couple other places with session scope I want to change, but i want to check how much it slows our tests down first. something for another PR.